.babelrc.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-react',
+      {
+        runtime: 'automatic',
+        importSource: 'preact',
+      },
+    ],
+    [
+      '@babel/preset-env',
+      {
+        bugfixes: true,
+        modules: false,
+        targets: {
+          chrome: '70',
+          firefox: '70',
+          safari: '11.1',
+          edge: '79',
```

Comment:
These targets have been changed to reflect https://github.com/hypothesis/client/issues/3744.